### ISSUE TITLE
train problem

### ISSUE THREAD

**Issue by Nine9Nine** [leeyoshinari/YOLO_v2#6]
I use your program to train my dataset（8 classes，2500pictures）,the loss is too huge,should I change any parameters？

**Comment by Nine9Nine** [comment]
![image](https://user-images.githubusercontent.com/33343930/40611190-ebcc058a-62a7-11e8-962c-93c488792ab1.png)

**Comment by leeyoshinari** [comment]
Are you train from scratch, or using pre-train model? Maybe you can use smaller learning rate. It maybe related to your dataset distribution.

**Comment by Nine9Nine** [comment]
我是从零开始训练的，把weights路径注释掉了，学习率调了还是爆炸sad

**Comment by leeyoshinari** [comment]
Because of the network is deep and difficult to train, my suggestion is  using a pre-training model. If you want to start training from scratch, you can freeze some layers during training.

**Comment by Nine9Nine** [comment]
多谢大佬

**Comment by Nine9Nine** [comment]
hello my lord, I convert the dataset to C version Yolo's format,the training process is smoothly.I think maybe something is wrong in your train program...

**Comment by ruikun** [comment]
@Nine9Nine  hello bro!
I got the same problem, the loss is exploding to nan.
How did you handle this problem? thanks a lot!

**Comment by Nine9Nine** [comment]
@ruikun 没解决，先用官网C版本对付了一下，之后还会继续研究

**Comment by ruikun** [comment]
@Nine9Nine  你好，感谢回复，能不能麻烦你看一看我新开的帖里提出的问题?
https://github.com/leeyoshinari/YOLO_v2/issues/7
帮忙给个思路，现在也不知道是train的方法有问题还是数据集的毛病。多谢!
